app/serializers/group_assignment_repo_serializer.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class GroupAssignmentRepoSerializer < ActiveModel::Serializer
+  attributes :id
+  attributes :username
+  attributes :repoUrl
+  attributes :displayName
+
+  def username
+    object.group.title
+  end
+
+  # rubocop:disable MethodName
```

Comment:
Returning in camel case to keep with JSON standards

Reply:
Nitpick: It's convention in Rails to use snake case in JSON. Take a look at GitHub's REST API for example https://developer.github.com/v3/

Reply:
I also used camel case here to make life on the Classroom-Desktop side (because everything there has to be camel case 🤦‍♂️ I think it makes sense to switch to snake case, but I'll do that as a separate PR in the future after finalizing the Classroom-Desktop code.